infrastructure/versions.tf

```diff
@@ -18,7 +18,8 @@ terraform {
       source = "hashicorp/google"
     }
     google-beta = {
-      source = "hashicorp/google-beta"
+      source  = "hashicorp/google-beta"
+      version = ">= 5.40.0, != 5.44.0, !=6.2.0"
```

Comment:
Leave a comment for while these versions are excluded

Reply:
Added